### ISSUE TITLE
Compatibility with Python 3.8 (asyncio.coroutine is deprecated)

### ISSUE THREAD

**Issue by szborows** [kblin/mockaioredis#14]
Hi,

it looks like `@asyncio.coroutine` was deprecated in Python 3.8 and will be removed in Python 3.10. Is there any reason why `async def` is not used in mockaioredis? I can imagine there is something related to mocking that isn't straightforward with `async def`, thus I'm asking before doing any work on this.

The errors coming from this project look like that:

```
/builds/[redacted]/.tox/py38/lib/python3.8/site-packages/mockaioredis/commands/set.py:129
  /builds/[redacted]/.tox/py38/lib/python3.8/site-packages/mockaioredis/commands/set.py:129: DeprecationWarning: "@coroutine" decorator is deprecated since Python 3.8, use "async def" instead
```

**Comment by kblin** [comment]
I think the main reason is that when I started, python 3.4 was still a supported version, and didn't yet have the `async def`. That should be updated.

**Comment by kblin** [comment]
Fixed in 0.0.16 that I just released.

**Comment by szborows** [comment]
thanks @kblin
